roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -35,6 +38,7 @@ netchecker_server_cpu_limit: 100m
 netchecker_server_memory_limit: 256M
 netchecker_server_cpu_requests: 50m
 netchecker_server_memory_requests: 64M
+netchecker_server_deployment_nodeselector: "kubernetes.io/os: linux"
```

Comment:
I could not find any places this netchecker_server_deployment_nodeselector is used in this PR.
It would be fine to remove this change if unnecessary.

Reply:
I wasn't mindful. Nice catch :) Thank a lot!